mailpoet/lib/EmailEditor/Integrations/MailPoet/EmailEditor.php

```diff
@@ -112,6 +113,25 @@ public function enqueueAssets(): void {
       [],
       $assetsParams['version']
     );
+
+    // Enqueue inline script with MailPoet specific editor settings
+    $jsonAPIRoot = rtrim($this->wp->escUrlRaw(admin_url('admin-ajax.php')), '/');
+    $token = $this->wp->wpCreateNonce('mailpoet_token');
+    $apiVersion = API::CURRENT_VERSION;
+    $inlineScript = <<<EOL
+var mailpoet_json_api_root = '%s';
+var mailpoet_token = '%s';
+var mailpoet_api_version = '%s';
+EOL;
+    $this->wp->wpAddInlineScript(
```

Comment:
As these are only static variables it would be nicer to use https://developer.wordpress.org/reference/functions/wp_localize_script/

Example useage in MailPoet: PHP: https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/Form/AssetsController.php#L101-L105
Types: https://github.com/mailpoet/mailpoet/blob/d2d465165dfb91d50854754fffcc789b98f23f89/mailpoet/assets/js/src/global.d.ts#L155C1-L158C5

Reply:
Nice suggestion. I will change it.